ape_alchemy/providers.py

```diff
@@ -47,6 +30,15 @@ class AlchemyEthereumProvider(Web3Provider, UpstreamProvider):
 
     @property
     def uri(self):
+        """
+         Your Alchemy RPC URI, including the project ID.
+
+        `ecosystem`: choose from supported ecosystems
+        `network`: choose a network to connect to
+
+        Returns:
```

Comment:
`Returns:` / `Args:` statements are not needed for properties

Reply:
Updated [3f4682c](https://github.com/ApeWorX/ape-alchemy/pull/23/commits/3f4682cdab7724cc3bc6f70a1c6a45ae5e91da74)